src/test/scala/sigmastate/lang/SigmaCompilerTest.scala

```diff
@@ -246,24 +246,20 @@ class SigmaCompilerTest extends SigmaTestingCommons with LangTests with ValueGen
   }
 
   property("Collection.indices") {
-    testMissingCosting("Coll(true, false).indices",
+    comp("Coll(true, false).indices") shouldBe
       mkMethodCall(
         ConcreteCollection(TrueLeaf, FalseLeaf),
         SCollection.IndicesMethod,
         Vector()
       )
-    )
   }
 
   property("SCollection.flatMap") {
-    testMissingCosting("OUTPUTS.flatMap({ (out: Box) => Coll(out.value >= 1L) })",
+    comp("OUTPUTS.flatMap({ (out: Box) => Coll(out.value >= 1L) })") shouldBe
```

Comment:
@greenhat The flatMap can accept only limited lambdas as arguments. 
Currently:
1) `x => x._1`
2) `box => MethodCall(box, name, List())`
3) `x: Coll[T] => x`

Reply:
please put this requirements into description of method call both LangSpec.md and special.collection.Colls.scala

Reply:
@aslesarenko Done. What's the reason for this limitation? I'm struggling to find where and how it's enforced.